### ISSUE TITLE
Test screen reader alerts and keyboard navigation for admin search options

### ISSUE THREAD

**Issue by kimisgold** [omeka/Omeka#1038]
This issue refers to the admin global search and its toggle-able options panel. There have been recent updates that affect this issue, so please ensure the test installation has the latest from `master`.

![image](https://github.com/user-attachments/assets/d8db40af-27a3-4825-b032-2bad9f5cf03b)

Test the following:

* The user can navigate to the options toggle button and activate it via keyboard.
* When the button activates, screen readers should read out an "expanded" or "collapsed" status change on the button.
* When the button is "expanded" the user should be able to tab to the search options panel. Tabbing past the options should land the user on the search submit button.
* When the button is "collapsed", the options panel should no longer be keyboard accessible. The user should be able to tab between the options and submit buttons.

Please test in Firefox, Chrome, Safari, Edge, and a mobile browser using JAWS, NVDA, Voiceover, and Talkback if possible.

**Comment by katknow** [comment]
This is working in every screen reader except for Voiceover on the MacOS. It is, however, working in iOS. I tested it in all browsers listed.

**Comment by kimisgold** [comment]
What is it doing in Voiceover?

**Comment by katknow** [comment]
It is just recognizing it as a button, not reading any of the expand, collapse, or options when you tab through.

**Comment by kimisgold** [comment]
I'm wondering if it's a Voiceover settings. Do you ever see those options for any other element?

This is what I see. The video does not have audio but does display the screen reader text.


https://github.com/user-attachments/assets/de7fbbb4-4b5b-4297-8f5d-8ccde7bf2769

**Comment by kimisgold** [comment]
I've been working with verbosity set to high in my Voiceover settings.

![image](https://github.com/user-attachments/assets/66a4ca89-3ad8-4683-a24b-ce95a48dbd61)

**Comment by katknow** [comment]
That was it! Changed the verbosity setting and it is now working as expected.
